Analysis/DataModel/include/Analysis/JetFinder.h

```diff
@@ -0,0 +1,234 @@
+                                                                                                               ghostRepeatN(1),
+                                                                                                               ghostktMean(1e-100), //is float precise enough?
+                                                                                                               gridScatter(1.0),
+                                                                                                               ktScatter(0.1),
+                                                                                                               jetBkgR(0.2),
+                                                                                                               bkgPhiMin(phi_Min),
+                                                                                                               bkgPhiMax(phi_Max),
+                                                                                                               bkgEtaMin(eta_Min),
+                                                                                                               bkgEtaMax(eta_Max),
+                                                                                                               algorithm(fastjet::antikt_algorithm),
+                                                                                                               recombScheme(fastjet::E_scheme),
+                                                                                                               strategy(fastjet::Best),
+                                                                                                               areaType(fastjet::active_area),
+                                                                                                               algorithmBkg(fastjet::JetAlgorithm(fastjet::kt_algorithm)),
+                                                                                                               recombSchemeBkg(fastjet::RecombinationScheme(fastjet::E_scheme)),
+                                                                                                               strategyBkg(fastjet::Best),
+                                                                                                               areaTypeBkg(fastjet::active_area),
+                                                                                                               bkgSubMode(BkgSubMode::none),
+                                                                                                               constSubAlpha(1.0),
+                                                                                                               constSubRMax(0.6),
+                                                                                                               isReclustering(false)
```

Comment:
consider Non-static member initialization instead

Reply:
We will build a dictionary for this class in the next step and expose the data members as configurable